Albania/ALB_extract_microdata_excel_to_csv.py

```diff
@@ -10,20 +10,21 @@
 COUNTRY = 'Albania'
 microdata_csv_dir = prepare_microdata_csv_dir(COUNTRY)
 filename = input_excel_filename(COUNTRY)
-disaggregated_data_sheets = ['Data_Expenditures', 'Data_Revenues']
+disaggregated_data_sheets = ['Data_Expenditures']
 
 for sheet in tqdm(disaggregated_data_sheets):
     csv_file_path = f'{microdata_csv_dir}/{sheet}.csv'
-    df = pd.read_excel(filename, sheet_name=sheet, header=0)
+    df = pd.read_excel(filename, sheet_name=sheet)
```

Comment:
If there's only one data sheet to be processed, consider removing the for loop and explicitly specify the sheet name here.